dotnet/content/scmApplicationHost.xdt

```diff
@@ -2,4 +2,16 @@
 <configuration xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
   <!-- This file exists to prevent applicationHost.xdt from being applied to scm host that runs
       other dotnet processes (dotnet build, csc etc. )-->
+	<environmentVariables xdt:Transform="InsertIfMissing">
+        
+        <add name="DD_APM_WINDOWS_PIPE_NAME" value="datadogtrace-uniqueTracePipeId" xdt:Locator="Match(name)" xdt:Transform="Insert"/>	
```

Comment:
Do you need to do the thing where you have `xdt:Transform="RemoveAll"` followed by `xdt:Transform="Insert"`?

Reply:
Not needed, but it's good practice and consistent.
I think I will re-org the application host to be more readable in another PR as well.
There are also some things we may not want to override if the user sets them.
Like disabling the profiler.

Reply:
Done